manifests/init.pp

```diff
@@ -146,6 +135,24 @@
     value    => $error_log,
   }
 
+  php::apache::config { 'php-memory-limit-server':
```

Comment:
What happens if server is not set to apache? We can merge this as is but we should do a follow up removing this assumption here (and elsewhere in the file).